common/lib/xmodule/xmodule/tests/test_vertical.py

```diff
@@ -83,10 +83,13 @@ class BaseVerticalBlockTest(XModuleXmlImportTest):
     """
     test_html = 'Test HTML'
     test_problem = 'Test Problem'
+    test_html_nested = 'Nest HTML'
+    test_problem_nested = 'Nest_Problem'
```

Comment:
Nit: For similarity, switching these to `Test Nested HTML` and `Test Nested Problem`

It doesn't matter at all though.

Reply:
Updated! I did leave underscores since some word wrapping was interfere with the string being found

Reply:
Oh that's bonkers, but okay!